orchagent/main.cpp

```diff
@@ -595,13 +603,14 @@ int main(int argc, char **argv)
 
         if (gMySwitchType == "voq" || gMySwitchType == "chassis-packet")
         {
-            attr.value.u64 = (5 * SAI_REDIS_DEFAULT_SYNC_OPERATION_RESPONSE_TIMEOUT);
+            redis_timeout = (5 * SAI_REDIS_DEFAULT_SYNC_OPERATION_RESPONSE_TIMEOUT);
         }
         else if (gMySwitchType == "fabric")
         {
-            attr.value.u64 = (10 * SAI_REDIS_DEFAULT_SYNC_OPERATION_RESPONSE_TIMEOUT);
+            redis_timeout = (10 * SAI_REDIS_DEFAULT_SYNC_OPERATION_RESPONSE_TIMEOUT);
         }
 
+        attr.value.u64 = redis_timeout*asan_delay_factor;
```

Comment:
I think we can simplify the implementation:
```
#ifdef ASAN_ENABLED
attr.value.u64 *= asan_delay_factor;
#endif
```
This will allow for not introducing variables that will be used in runtime when ASAN is not enabled

Reply:
You cannot avoid asan_enabled or delay_factor. Else we may need to duplicate some code. Removing redis_timeout